src/main/groovy/uk/ac/sanger/scgcf/jira/services/actions/TransferActions.groovy

```diff
@@ -37,9 +59,22 @@ class TransferActions {
         destinationLabwares[0]
     }
 
+    /**
+     * A stamp from a single {@code Labware} into a single {@code Labware}.
```

Comment:
This documentation seems to be the same as the 'normal stamp'. We should add the difference.
You have to add the specific locations to transfer for the selective stamp, otherwise it won't transfer anything.